dbt_project.yml

```diff
@@ -1,5 +1,5 @@
 name: 'the_tuva_project'
-version: '0.10.8'
```

Comment:
We cannot accept this change as it will affect the version other users install. Please remove this commit from your PR.